pages/socios/crear.tsx

```diff
@@ -22,7 +22,16 @@ const Create: NextPage = () => {
         position: 'Socio',
         type: 'MEMBER',
       }
-      await axios.post('/api/members', member)
+      const user = await axios.post('/api/members', member)
+      const { data } = user.data
+      const { startDate, memberId } = data
+      const parseStartDate = new Date(startDate)
+      const expiration = new Date(parseStartDate.setFullYear(parseStartDate.getFullYear() + 1))
+      const subscriptionData = {
+        expiration: expiration.toLocaleDateString(),
+        memberId,
+      }
+      await axios.post('/api/subscriptions', subscriptionData)
```

Comment:
add a guard here

Reply:
And in this case, would it be to store in a var and validate that variable, throwing an error in case of failure?

Reply:
Sounds like a plan. Also, this returns an error like

`const { data, error } = await axios.post('/api/subscriptions', subscriptionData)`

The recommended way it's using chined resolvers like here https://axios-http.com/docs/handling_errors

Another ref: https://dev.to/mperon/axios-error-handling-like-a-boss-333d